content/docs/learning-guide/tutorials/extend-the-editor/shape-example-py.md

```diff
@@ -0,0 +1,508 @@
+---
+linkTitle: Create a Custom Tool Gem in Python
+title: Create a Custom Tool Gem in Python to Extend Open 3D Engine Editor
+description: Learn how to extend the Open 3D Engine (O3DE) Editor by creating a custom tool Gem in Python.
+weight: 300
+toc: true
+---
+
+In this tutorial, you create a custom tool Gem written in Python, called **Shape Example**, that extends the **Open 3D Engine (O3DE) Editor**. The Shape Example tool allows you to create entities with a Shape component and configure their component properties. You'll learn how to use the `PythonToolGem` template and practice Python development with [Qt](https://wiki.qt.io/Main), O3DE Tools UI API, and other O3DE APIs.
+
+This tutorial shows you how to create the **PyShapeExample** Gem, which is a sample Gem in [`o3de/sample-code-gems` repository](https://github.com/o3de/sample-code-gems/tree/main/py_gems/PyShapeExample). You can reference the PyShapeExample Gem sample as you follow along this tutorial.
+
+By the end of this tutorial, you'll be able to extend the Editor by creating your own custom tools written in C++.
+
+The following image is a preview of the Shape Exmple tool that you create in this tutorial.
+
+{{< image-width "/images/learning-guide/tutorials/extend-the-editor/shape-example/python-shape-example-demo.png" "1080" "An image of the Shape Example tool and some entities created by it." >}}
+
+
+## Prerequisites
+
+Before you start the tutorial, ensure that you have the following:
+
+- Set up an O3DE development environment. You can build O3DE as a source engine or a pre-built SDK engine. For more information, refer to [Setting up O3DE from GitHub](/docs/welcome-guide/setup/setup-from-github.md).
+
+
+## Create a Gem with the `PythonToolGem` template
+
+To start, create a Gem by using the `PythonToolGem` template. The `PythonToolGem` template provides a basic Python framework to create a dialog window that extends the Editor.
+
+This tutorial uses the following directory names and locations in the examples.
+
+- **O3DE engine directory**: `C:/o3de`  
```

Comment:
Same comment from the C++ tutorial about consistent use of back slashes applies to this Python tutorial.